tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -60,6 +60,10 @@ def cluster_keys(cluster):
     return cluster.keys()
 
 
+def cluster_config():
```

Comment:
what is this for

Reply:
It's for the test below which needs to retrieve the cluster_config from on the cluster